internal/controllers/topology/cluster/reconcile_state.go

```diff
@@ -198,13 +199,13 @@ func (r *Reconciler) reconcileControlPlane(ctx context.Context, s *scope.Scope)
 		},
 		)
 		if err != nil {
-			return errors.Wrapf(err, "failed to update %s", tlog.KObj{Obj: s.Desired.ControlPlane.InfrastructureMachineTemplate})
+			return errors.Wrapf(err, "failed to reconcile %s", tlog.KObj{Obj: s.Desired.ControlPlane.InfrastructureMachineTemplate})
```

Comment:
Do we also only have to log the kind here?

Reply:
Same as: https://github.com/kubernetes-sigs/cluster-api/pull/5971#discussion_r791983822